newrelic-slack-ruby-bot.gemspec

```diff
@@ -14,5 +14,5 @@ Gem::Specification.new do |s|
   s.licenses = ['MIT']
   s.summary = 'NewRelic instrumentation for slack-ruby-bot.'
   s.add_dependency 'newrelic_rpm'
-  s.add_dependency 'slack-ruby-bot', '<= 0.7.0'
+  s.add_dependency 'slack-ruby-bot', '> 0.7.0'
```

Comment:
This should be >= the exact version with which it works, so if 0.8.0 was the first one where this change shipped, it should be >= 0.8.0.